core/modules/savers/override.js

```diff
@@ -0,0 +1,58 @@
+/*\
+title: $:/core/modules/savers/override.js
+type: application/javascript
+module-type: saver
+
+Looks for `window.tiddlyWikiOverrides.saver` first on the current window, then
+on the parent window (of an iframe). If present, the saver must define
+	save: function(text,method,callback) { ... }
+and the saver may define
+	priority: number
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+var findSaver = function(window) {
+	try {
+		return window && window.tiddlyWikiOverrides && window.tiddlyWikiOverrides.saver;
```

Comment:
I put the catch-all in because I wasn't sure what browsers do when the iframe comes from a different origin. I just did a test with Firefox and this line of code `if (window.parent && window.parent.foo)` throws this exception `Uncaught DOMException: Permission denied to access property "foo" on cross-origin object`. I don't know of any easy way to detect whether an iframe is cross-origin. So I think swallowing and logging the exception is safest here, especially considering the risk of introducing a fatal error on startup. I'll clarify the comment now that I've confirmed the behavior.

Some links that seem to indicate try/catch is the best we can do: https://stackoverflow.com/q/2365822 and https://stackoverflow.com/q/12381334

I agree with the rest of your feedback and will make the changes. Thanks for your consideration.